wit/wasi-nn.wit

```diff
@@ -127,7 +127,7 @@ interface inference {
     compute: func(ctx: graph-execution-context) -> result<_, error>;
 
     /// Extract the outputs after inference.
-    get-output: func(ctx: graph-execution-context, index: u32) -> result<tensor-data, error>;
+    get-output: func(ctx: graph-execution-context, name: string) -> result<tensor-data, error>;
```

Comment:
This should probably return tensor and not tensor data, but is outside the scope of the changes in this PR>